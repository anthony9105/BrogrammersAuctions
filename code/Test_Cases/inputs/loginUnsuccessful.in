login randomUser1 dswibd