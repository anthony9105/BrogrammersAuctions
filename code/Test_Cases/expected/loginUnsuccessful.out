Error. Login unsuccessful.  No account corresponding to this username